### PR TITLE
Include vars for ansible version >= 2.6

### PR DESCRIPTION
**Pull request by djzager**

### CODE REVIEW

**Comment by jmontleon** [comment]
ACK

File: templates/service.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 
-apiVersion: v1
```
Comment (fabianvf) [inline templates/service.yaml]: This one is always v1, don't think you need a parameter

File: templates/pvc.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 
-apiVersion: v1
+apiVersion: {{ pvc_api_version }}
```
Comment (fabianvf) [inline templates/pvc.yaml]: This one is always v1, don't think you need a parameter

File: tasks/main.yml [hunk 3]
```diff
@@ -14,6 +14,10 @@
   set_fact:
     cluster: "{{ 'openshift' if 'openshift' in oc_version.stdout else 'kubernetes' }}"
 
+- name: 'Include variables based on ansible version'
+  include_vars: ansible_26.yml
```
Comment (fabianvf) [inline tasks/main.yml]: Do you need to import the 25 variables as well?

File: templates/deployment.yaml [hunk 4]
```diff
@@ -51,12 +52,12 @@ spec:
...
         ports:
-          - container_port: 8080
+          - containerPort: 8080
             protocol: TCP
         volumeMounts:
           - name: mediawiki
             mountPath: /persistent
       volumes:
         - name: mediawiki
-          persistent_volume_claim:
```
Comment (fabianvf) [inline templates/deployment.yaml]: Will these changes work with v1?

File: tasks/main.yml (hunk 3 above)
Comment (djzager) [reply tasks/main.yml]: I just use them to override 25 variables.

File: templates/deployment.yaml (hunk 4 above)
Comment (djzager) [reply templates/deployment.yaml]: I tested using 2.5 and it still worked.

**Pull request merged**